tests/PostApiTest.php

```diff
@@ -5,6 +5,8 @@
 
 class PostApiTest extends TestCase
 {
+    use WithoutMiddleware;
```

Comment:
We made a [`withHeader` helper function](https://github.com/DoSomething/northstar/blob/49630b5f11f4f5c8d0a6ad4a63097f62f8b48548/tests/TestCase.php#L283-L296) for that in Northstar if you wanna steal the approach. 😄